f5/bigip/sys/test/test_dns.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import mock
+import pytest
+
+from f5.bigip.mixins import UnnamedResourceMixin
+from f5.bigip.sys.dns import Dns
+
+
+@pytest.fixture
+def FakeDns():
+    fake_sys = mock.MagicMock()
+    return Dns(fake_sys)
+
+
+class TestDns(object):
```

Comment:
We don't need a class scope here.   Let's remove `class TestDns` and "promote" its methods to module-level functions.